### PR TITLE
fix some build warnings

### PR DESCRIPTION
**Pull request by Onvember**

### CODE REVIEW

File: doc/reference/reference_lua/digest.rst [hunk 1]
```diff
@@ -305,11 +306,12 @@ Below is a list of all ``digest`` functions.
 
     Returns a number made with consistent hash.
 
-    The guava function uses the `Consistent Hashing <https://en.wikipedia.org/wiki/Consistent_hashing>`_
-    algorithm of the Google
-    guava library. The first parameter should be a hash code; the second
-    parameter should be the number of buckets; the returned value will be an
-    integer between 0 and the number of buckets. For example,
+    The guava function uses the
```
Comment (veod32) [inline doc/reference/reference_lua/digest.rst:309]:
```suggestion
    The ``guava()`` function uses the
```

File: doc/reference/reference_lua/digest.rst [hunk 2]
```diff
@@ -305,11 +306,12 @@ Below is a list of all ``digest`` functions.
...
+    `Consistent Hashing <https://en.wikipedia.org/wiki/Consistent_hashing>`_
+    algorithm of the Google guava library. The first parameter should be a hash
```
Comment (veod32) [inline doc/reference/reference_lua/digest.rst:311]: Can we put an external link to the doc or any other reference about this library?

File: doc/reference/reference_lua/digest.rst [hunk 3]
```diff
@@ -305,11 +306,12 @@ Below is a list of all ``digest`` functions.
...
+    code; the second parameter should be the number of buckets;
+    the returned value will be an integer between 0 and the number of buckets.
```
Comment (veod32) [inline doc/reference/reference_lua/digest.rst:313]: Suggest to re-write the entire paragraph (starting from "The first parameter...) using options `:param:` and `:return:` which we usually use together with the `.. function::` directive.

File: doc/reference/reference_lua/digest.rst (hunk 2 above)
Comment (Onvember) [reply doc/reference/reference_lua/digest.rst]: noted it in https://github.com/tarantool/doc/issues/2019

File: doc/reference/reference_lua/digest.rst (hunk 3 above)
Comment (Onvember) [reply doc/reference/reference_lua/digest.rst]: noted it in https://github.com/tarantool/doc/issues/2019

File: doc/reference/reference_lua/digest.rst (hunk 1 above)
Comment (Onvember) [reply doc/reference/reference_lua/digest.rst]: TBD in https://github.com/tarantool/doc/issues/2019

**Comment by NickVolynkin** [comment]
@Onvember good job )

**Pull request merged**